_source/logzio_collections/_log-sources/aws-control-tower.md

```diff
@@ -0,0 +1,173 @@
+| `logType`        | The log type you'll use with this Lambda. This is shown in your logs under the type field in Kibana. Logz.io applies parsing based on the log type.                                                                                                                 | Default: `s3_hook` |
+| `pathsRegexes`   | Comma-seperated list of regexes that match the paths you'd like to pull logs from.                                                                                                                                                                                  | -                  |
+| `pathToFields`   | Fields from the path to your logs directory that you want to add to the logs. For example, `org-id/aws-type/account-id` will add each of the fields `ord-id`, `aws-type` and `account-id` to the logs that are fetched from the directory that this path refers to. | -                  |
+
+
+##### Add trigger
+
+Give the stack a few minutes to be deployed.
+
+Once your Lambda function is ready, you'll need to manually add a trigger. This is due to Cloudformation limitations.
+
+Go to the function's page, and click on **Add trigger**.
+
+![Step 5 screenshot](https://dytvr9ot2sszz.cloudfront.net/logz-docs/control-tower/s3-hook-stack-05.png)
+
+Then, choose **S3** as a trigger, and fill in:
+
+- **Bucket**: Your bucket name.
+- **Event type**: Choose option `All object create events`.
+- Prefix and Suffix should be left empty.
+
+Confirm the checkbox, and click **Add*.
+
+![Step 5 screenshot](https://dytvr9ot2sszz.cloudfront.net/logz-docs/control-tower/s3-hook-stack-06.png)
+
+
+##### Filtering files
+
+If there are specific paths within the bucket that you want to pull logs from, you can use the `pathsRegex` variable.
+This variable should hold a comma-seperated list of regexes that match the paths you wish to extract logs from.
+**Note**: This will still trigger your Lambda function every time a new object is added to your bucket. However, if the key does not match the regexes, the function will quit and won't send the logs.
+
+
+##### Adding object path as logs field
+
+In case you want to use your objects' path as extra fields in your logs, you can do so by using `pathToFields`.
+
+For example, if your objects are under the path: `oi-3rfEFA4/AWSLogs/2378194514/file.log`, where `oi-3rfEFA4` is org id, `AWSLogs` is aws type, and `2378194514` is account id. 
+
+Setting `pathToFields` with the value: `org-id/aws-type/account-id` will add to logs the following fields:
+`org-id`: `oi-3rfEFA4`, `aws-type`: `AWSLogs`, `account-id`: `2378194514`.
+
+<!-- info-box-start:info -->
+If you use `pathToFields`, you need to add a value for each subfolder in the path. Otherwise there will be a mismatch and the logs will be sent without fields.
+{:.info-box.note}
+<!-- info-box-end -->
+
+**Important notes about `pathToFields`**:
+
```

Comment:
These aren't steps, but more of side notes for people who want to use these features. They can be in a separate tab of "advanced settings" or something like that? Just so that the instruction wouldn't appear that long.